Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -179,20 +179,16 @@
         return;
     }
 
-    if (state != WebTextReplacementData::State::Committed && state != WebTextReplacementData::State::Reverted && state != WebTextReplacementData::State::Active)
```

Comment:
Nit: Let's just avoid the replacement marker finding and simple range construction work by keeping the early return?

Reply:
`state` will never not be these anyways, so the `if` statement is unnecessary